frontend/src/components/BlockDetails/BlockDetailsHeader.vue

```diff
@@ -15,6 +15,16 @@
 					icon-size="h-5 w-5"
 					tooltip-class="font-sans"
 				/>
+				<h1 class="inline-block text-2xl numerical">
+					{{ String(block?.id) }}
+				</h1>
+				<TextCopy
+					:text="String(block.id)"
```

Comment:
I think this should be using `block.blockHeight` instead of `block.id`.
The ID type in graphql is not meant to convey information, but is simply an identifier for the resource.

Reply:
Also the .NET backend does not use block height as `block.id`